jose-jwt/crypto/ConcatKDF.cs

```diff
@@ -34,14 +34,41 @@ public static byte[] DeriveKey(CngKey externalPubKey, CngKey privateKey, int key
                             if (status != BCrypt.ERROR_SUCCESS)
                                 throw new CryptographicException(string.Format("NCrypt.NCryptDeriveKey() failed with status code:{0}", status));
 
-                            return Arrays.LeftmostBits(secretKey, keyBitLength);
+                            return Arrays.LeftmostBits(secretKey, keyBitLength);
                         }
                     }
                 }
             }
-#else
+#else
             throw new NotImplementedException("not yet");
-#endif
+#endif
+        }
+        
+        public static byte[] DeriveKeyNonCng(ECDiffieHellman externalPubKey, ECDiffieHellman privateKey, int keyBitLength, byte[] algorithmId, byte[] partyVInfo, byte[] partyUInfo, byte[] suppPubInfo)
```

Comment:
funny name :)

Reply:
hahaha true. I'll change to make it more explicit, like DeriveEcdhKey